accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/helper/ContentHelper.java

```diff
@@ -77,8 +77,7 @@ public static boolean createInitialContent(final Session session, final Installa
                 history.addMessage(LOG, "Created initial content for path " + path);
                 return true;
             } catch (Exception e) {

```

Comment:
distinguish between PathNotFoundException and everything else. The first one should emit a WARN only.